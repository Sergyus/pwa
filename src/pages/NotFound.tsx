export default function NotFound(): JSX.Element {
  return (
    <div style={{ marginTop: '20px' }}>
      <h1>404 ERROR!</h1>
      <div style={{ fontSize: '24px', marginBottom: '5px' }}>
        Page not found :(
      </div>
      <p>
        Maybe the page you are looking for has been removed, or you typed in the
        wrong URL
      </p>
    </div>
  );
}
