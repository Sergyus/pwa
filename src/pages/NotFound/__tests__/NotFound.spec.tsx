import React from 'react';
import { render } from '@testing-library/react';
import NotFound from '../NotFound';

describe('<NotFound />', () => {
  it('should renders', () => {
    const { asFragment } = render(<NotFound />);
    expect(asFragment()).toMatchSnapshot();
  });
});
