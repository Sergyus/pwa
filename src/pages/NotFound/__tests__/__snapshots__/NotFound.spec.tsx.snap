// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound /> should renders 1`] = `
<DocumentFragment>
  <div
    style="margin-top: 20px;"
  >
    <h1>
      404 ERROR!
    </h1>
    <div
      style="font-size: 24px; margin-bottom: 5px;"
    >
      Page not found :(
    </div>
    <p>
      Maybe the page you are looking for has been removed, or you typed in the wrong URL
    </p>
  </div>
</DocumentFragment>
`;
